api/envoy/api/v2/route/route.proto

```diff
@@ -188,9 +188,11 @@ message WeightedCluster {
     // entries in the clusters array must add up to the total_weight, which defaults to 100.
     google.protobuf.UInt32Value weight = 2;
 
-    // Optional endpoint metadata match criteria. Only endpoints in the upstream
-    // cluster with metadata matching that set in metadata_match will be
-    // considered. The filter name should be specified as *envoy.lb*.
+    // Optional endpoint metadata match criteria used by the subset load balancer. Only endpoints in
+    // the upstream cluster with metadata matching what is set in this field will be considered.
```

Comment:
nit: will be considered *for load balancing*.

very small but might as well if you're already here. Same below.